packages/sign_in_with_apple/ios/Classes/SwiftSignInWithApplePlugin.swift

```diff
@@ -0,0 +1,68 @@
+import Flutter
+import UIKit
+import AuthenticationServices
+
+public class SwiftSignInWithApplePlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(name: "de.aboutyou.mobile.app.sign_in_with_apple", binaryMessenger: registrar.messenger())
+    let instance = SwiftSignInWithApplePlugin()
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    NSLog("NS receiving call \(call.method)")
+    print("receiving call \(call.method)");
+    
+    if #available(iOS 13.0, *) {
+        if call.method == "signInWithApple" {
+            let appleIDProvider = ASAuthorizationAppleIDProvider()
+            let request = appleIDProvider.createRequest()
+            request.requestedScopes = [.fullName, .email]
+            
+            let passwordProvider = ASAuthorizationPasswordProvider()
+            let passwordRequest = passwordProvider.createRequest()
+            
+            let authorizationController = ASAuthorizationController(authorizationRequests: [passwordRequest, request])
+            authorizationController.delegate = self
+            authorizationController.performRequests()
```

Comment:
@tp you will need to call here `result` to complete the method, otherwise, the call on the flutter side will never complete

Reply:
We also probably want to return some sort of status or error so the UI can react to it